### ISSUE TITLE
Crashes when clicked on - Artix (arch-based) Linux

### ISSUE THREAD

**Issue by peepo5** [wineasio/wineasio#26]
I am using playonwine so I think the install steps were not configurated for it, however wineasio appears in FL so i must have set it up mostly correct, however it still crashes.

The comment I left on another issue is below detailing the process. If I record a video I will post it as a comment.

I get error while accessing the asio driver then it immediately crashes. using 6.11 staging in playonlinux
like when i change to wineasio, it immediately dies.
I installed the AUR on top of my manual install and i added the user to the group. Seems to have made the crash less worse, but it still crashes FL

commands I ran:
```bash
git clone --recursive https://github.com/wineasio/wineasio.git winea
cd winea

paru -S jack
paru -S jack2
paru -S lib32-jack2

make 32
sudo cp build32/wineasio.dll.so ~.PlayOnLinux/wineprefix/FL_STUDIO/drive_c/windows/system/wineasio.dll
regsvr32 ~.PlayOnLinux/wineprefix/FL_STUDIO/drive_c/windows/system/wineasio.dll
env WINEPREFIX=~/.PlayOnLinux/wineprefix/FL_STUDIO/ regsvr32 ~/.PlayOnLinux/wineprefix/FL_STUDIO/drive_c/windows/system/wineasio.dll

paru -S wineasio
sudo usermod -a -G realtime admin #fixed glitches when crashing but app still crashes
```

Running artix kernel 5.12.14-artix1-1
Based on arch
Using FL Studio 12.1.2 32bit
playonlinux wine version: wine-6.11-staging
terminal wine version: wine-6.12

I have some ideas to why it is not working.
The regsvr32 is for my terminal wine not playonlinux wine?
The wineprefix is in the wrong directory?

I would be highly grateful if i could get some help running wineasio on playonlinux or at all tbh

**Comment by szszoke** [comment]
I might be having the same issue.

Whenever I try to select ASIO in an application it ends in a crash.

There is no error message except this one:

```
**** stack smashing detected ***
```

I am running PipeWire with a Jack dropin package so that all the Jack server libraries are provided by PipeWire.

I am running Wine 6.20 on Kernel 5.15.

I compiled the latest comit on master and then registered the DLL successfully.

I was able to reproduce the issue both with Ableton Live 11 Standard and [ASIO Driver Tester](https://forum.vb-audio.com/viewtopic.php?t=1204).

I was not able to test without PipeWire so far.
